src/lib/artefact.mli

```diff
@@ -10,6 +10,8 @@ module Descr : sig
     | Ranking
     | Yans of { criterion : string list; }
   [@@deriving yojson]
+  (** Description of artefact types.
+      Enable reading of artefacts.*)
```

Comment:
I don't think we need that last line; the types of the values later to decode artefacts will make it clear.